jscomp/syntax/ast_external_attributes.ml

```diff
@@ -650,7 +650,7 @@ let handle_attributes
 
       | {set_index = true; _}
         ->
-        Bs_syntaxerr.err loc Conflict_ffi_attribute
+        Bs_syntaxerr.err loc (Conflict_ffi_attribute "You used an FFI attribute that can't be used with [@@set_index]")
```

Comment:
That is shorter and simpler, I like it. How about "Attribute found that conflicts with" - I used the singular because there could be just one conflicting attribute and used present tense for "conflicts" because the conflict is timeless.